sites/svelte-5-preview/src/lib/workers/compiler/index.js

```diff
@@ -127,3 +132,24 @@ function compile({ id, source, options, return_ast }) {
 		};
 	}
 }
+
+/** @param {import("../workers").MigrateMessageData} param0 */
+function migrate({ id, source }) {
+	try {
+		source = svelte.migrate(source);
+
+		return {
+			id,
+			source
+		};
```

Comment:
feels weird to return `id` here. what if it was a `{ code: string }` object (which would provide space for future additions like `metadata` or whatever)

Reply:
AFAIK it's necessary to relate the message? Don't know, left it in, but made the `code: string` change